auto_update_tests.py

```diff
@@ -84,121 +86,116 @@ def update_wasm_opt_tests():
     open(wast, 'w').write(open('a.wast').read())
 
     print('\n[ checking wasm-opt parsing & printing... ]\n')
-    for t in sorted(os.listdir(os.path.join(options.binaryen_test, 'print'))):
-        if t.endswith('.wast'):
-            print('..', t)
-            wasm = os.path.basename(t).replace('.wast', '')
-            cmd = WASM_OPT + [os.path.join(options.binaryen_test, 'print', t), '--print', '-all']
-            print('    ', ' '.join(cmd))
-            actual = subprocess.check_output(cmd)
-            print(cmd, actual)
-            with open(os.path.join(options.binaryen_test, 'print', wasm + '.txt'), 'wb') as o:
-                o.write(actual)
-            cmd = WASM_OPT + [os.path.join(options.binaryen_test, 'print', t), '--print-minified', '-all']
-            print('    ', ' '.join(cmd))
-            actual = subprocess.check_output(cmd)
-            with open(os.path.join(options.binaryen_test, 'print', wasm + '.minified.txt'), 'wb') as o:
-                o.write(actual)
+    for t in get_tests(PRINT_TEST_DIR, ['wast']):
```

Comment:
How about including the leading `.` in the extension list?  When I read `.wast` its more clear that its an extension.  Also, python in general extension tend in include the `.` (e.g. `os.path.splitext()`)

Reply:
Done

Reply:
Done